src/components/Game/Game.tsx

```diff
@@ -31,6 +66,12 @@ export const Game = ({ game }: GameProps) => {
 
   return (
     <div>
+      <Toaster
```

Comment:
Toaster is not relative to the lettergrid but fixed to window.
Wrap lettergrid wrapper in relative div and put Toaster in there. ContainerStyle of Toaster needs to changed to position absolute and offset adapted.